vlib/v/checker/tests/option_in_receiver_err.out

```diff
@@ -0,0 +1,7 @@
+vlib/v/checker/tests/option_in_receiver_err.vv:10:11: error: unexpected token `?`
+    8 | }
+    9 | 
+   10 | fn (mut f ?Foo) t1(arr ?[]int) ?string {
```

Comment:
I think an explicit message like `option types cannot have methods` will be better.

Reply:
Done.